htlcswitch/interfaces.go

```diff
@@ -60,14 +62,19 @@ type packetHandler interface {
 type dustHandler interface {
 	// getDustSum returns the dust sum on either the local or remote
 	// commitment.
-	getDustSum(remote bool) lnwire.MilliSatoshi
+	getDustSum(remote bool,
+		fee fn.Option[chainfee.SatPerKWeight]) lnwire.MilliSatoshi
 
 	// getFeeRate returns the current channel feerate.
 	getFeeRate() chainfee.SatPerKWeight
 
 	// getDustClosure returns a closure that can evaluate whether a passed
 	// HTLC is dust.
 	getDustClosure() dustClosure
+
+	// getCommitFee returns the commitment fee in satoshis from either the
+	// local or remote commitment.
+	getCommitFee(remote bool) btcutil.Amount
```

Comment:
I think the main question I have at this point is what the formal distinction is between the commit fees and the dust. It seems like starting with this commit, you want to redefine dust to be the commitment fees themselves.

Reply:
the commit fees here is the `CommitFee` field on the commitment struct which doesn't include dust. can update the comment to explicitly state that